web/src/components/RunDetailLayout/HeaderRight.tsx

```diff
@@ -9,21 +9,25 @@ import {useTestRun} from 'providers/TestRun/TestRun.provider';
 import {useTestSpecs} from 'providers/TestSpecs/TestSpecs.provider';
 import GuidedTourService, {GuidedTours} from '../../services/GuidedTour.service';
 import * as S from './RunDetailLayout.styled';
+import {useAppSelector} from '../../redux/hooks';
+import {selectIsPending} from '../../redux/testOutputs/selectors';
 
 interface IProps {
   testId: string;
   testVersion: number;
 }
 
 const HeaderRight = ({testId, testVersion}: IProps) => {
-  const {isDraftMode} = useTestSpecs();
+  const {isDraftMode: isTestSpecsPending} = useTestSpecs();
+  const isTestOutputsPending = useAppSelector(selectIsPending);
```

Comment:
Shouldn't be good to have a provider for outputs similar to what we have for test specs? or maybe a test draft provider which uses both 👀

Reply:
Done 👍  Moved that state to the provider. I have mixed feelings with adding the state (`outputs`) to the provider but it's ok. We should have consistency with the `TestSpecs` one. 

And totally agree with your idea of one single `test draft` provider. I didn't want to refactor test specs as part of this PR but I can work on that refactor in a new PR.